app/assets/stylesheets/general/_form.scss

```diff
@@ -312,3 +316,75 @@ input[type="file"] {
   font-size: $font-size-small;
   color: $text-color-dark;
 }
+
+.form-tags-wrapper {
+  display: flex;
+  flex-wrap: wrap;
+  align-items: center;
+  padding: 0 5px 5px 5px;
```

Comment:
This is more of a personal preference thing, but I've been avoiding using pixels in favor of rem. There's 2 advantages;
- Flexibility; If we ever decide to change the base font size everything else will scale accordingly.
- Consistency. A rem value is easier to relate to something else, given you use nice round values. For example it's easier to understand why you might use .5rem than it is why you might use 8px. 

In this case of course 5px is nice and round, but other values are not. I'd suggest going with .25rem here.

If you disagree please let me know!